plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -217,7 +249,11 @@ var JobListWidget = View.extend({
                 triggerJobClick: this.triggerJobClick,
                 JobStatus: JobStatus,
                 formatDate: formatDate,
-                DATE_SECOND: DATE_SECOND
+                DATE_SECOND: DATE_SECOND,
+                jobCheckedStates: this.jobCheckedStates,
+                jobHighlightStates: this.jobHighlightStates,
+                anyJobChecked: _.find(_.values(this.jobCheckedStates), (status) => status === true),
```

Comment:
There's no need to use `_.values` here, Underscore's [`_.find`](http://underscorejs.org/#find) works just fine on objects:
```
anyJobChecked: _.find(this.jobCheckedStates, (status) => status === true),
```